Games/Retro_ML.SuperMarioKart/Neural/Scoring/CollisionScoreFactor.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Retro_ML.Game;
```

Comment:
Modify the lines 91-96, shouldn't use Extra Field

https://github.com/Tourmi/Retro-ML/pull/27/files#diff-a63ddba70ba8370fe21362bb329039c96722031a5d1604c7a9f373dfcbb28e57L64-L69